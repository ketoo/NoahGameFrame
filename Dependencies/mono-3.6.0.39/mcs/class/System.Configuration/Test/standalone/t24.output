Rsa..ctor
   at System.Environment.GetStackTrace(Exception e, Boolean needFileInfo)
   at System.Environment.get_StackTrace()
   at RsaPassthroughProvider..ctor()
   at System.RuntimeTypeHandle.CreateInstance(RuntimeType type, Boolean publicOnly, Boolean noCheck, Boolean& canBeCached, RuntimeMethodHandle& ctor, Boolean& bNeedSecurityCheck)
   at System.RuntimeType.CreateInstanceSlow(Boolean publicOnly, Boolean fillCache)
   at System.RuntimeType.CreateInstanceImpl(Boolean publicOnly, Boolean skipVisibilityChecks, Boolean fillCache)
   at System.Activator.CreateInstance(Type type, Boolean nonPublic)
   at System.Configuration.TypeUtil.CreateInstanceWithReflectionPermission(Type type)
   at System.Configuration.ProtectedConfigurationSection.CreateAndInitializeProviderWithAssert(Type t, ProviderSettings pn)
   at System.Configuration.ProtectedConfigurationSection.InstantiateProvider(ProviderSettings pn)
   at System.Configuration.ProtectedConfigurationSection.GetProviderFromName(String providerName)
   at System.Configuration.BaseConfigurationRecord.GetProtectionProviderFromName(String providerName, Boolean throwIfNotFound)
   at System.Configuration.SectionInformation.ProtectSection(String protectionProvider)
   at T1.Main(String[] args)
Rsa.Initialize
   at System.Environment.GetStackTrace(Exception e, Boolean needFileInfo)
   at System.Environment.get_StackTrace()
   at RsaPassthroughProvider.Initialize(String name, NameValueCollection configurationValues)
   at System.Configuration.ProtectedConfigurationSection.CreateAndInitializeProviderWithAssert(Type t, ProviderSettings pn)
   at System.Configuration.ProtectedConfigurationSection.InstantiateProvider(ProviderSettings pn)
   at System.Configuration.ProtectedConfigurationSection.GetProviderFromName(String providerName)
   at System.Configuration.BaseConfigurationRecord.GetProtectionProviderFromName(String providerName, Boolean throwIfNotFound)
   at System.Configuration.SectionInformation.ProtectSection(String protectionProvider)
   at T1.Main(String[] args)
Rsa.Encrypt
   at System.Environment.GetStackTrace(Exception e, Boolean needFileInfo)
   at System.Environment.get_StackTrace()
   at RsaPassthroughProvider.Encrypt(XmlNode node)
   at System.Configuration.ProtectedConfigurationSection.EncryptSection(String clearXml, ProtectedConfigurationProvider provider)
   at System.Configuration.Internal.InternalConfigHost.System.Configuration.Internal.IInternalConfigHost.EncryptSection(String clearTextXml, ProtectedConfigurationProvider protectionProvider, ProtectedConfigurationSection protectedConfigSection)
   at System.Configuration.Internal.DelegatingConfigHost.EncryptSection(String clearTextXml, ProtectedConfigurationProvider protectionProvider, ProtectedConfigurationSection protectedConfigSection)
   at System.Configuration.Internal.DelegatingConfigHost.EncryptSection(String clearTextXml, ProtectedConfigurationProvider protectionProvider, ProtectedConfigurationSection protectedConfigSection)
   at System.Configuration.MgmtConfigurationRecord.GetConfigDefinitionUpdates(Boolean requireUpdates, ConfigurationSaveMode saveMode, Boolean forceSaveAll, ConfigDefinitionUpdates& definitionUpdates, ArrayList& configSourceUpdates)
   at System.Configuration.MgmtConfigurationRecord.SaveAs(String filename, ConfigurationSaveMode saveMode, Boolean forceUpdateAll)
   at System.Configuration.Configuration.SaveAsImpl(String filename, ConfigurationSaveMode saveMode, Boolean forceSaveAll)
   at System.Configuration.Configuration.SaveAs(String filename, ConfigurationSaveMode saveMode)
   at T1.Main(String[] args)
Section connectionStrings is now protected by MyProvider
