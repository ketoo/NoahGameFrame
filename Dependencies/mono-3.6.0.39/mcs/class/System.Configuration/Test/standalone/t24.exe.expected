The protection provider 'FooProvider' was not found.
